apps/web/app/queries/sbtc/sbtc-limits.query.ts

```diff
@@ -0,0 +1,64 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { z } from 'zod';
+import { useLeatherConnect } from '~/store/addresses';
+
+import { getStacksAssetStringParts } from '@leather.io/stacks';
+
+import { useStacksClient } from '../stacks/stacks-client';
+
+const emilyApiUrl = 'https://sbtc-emily.com';
+const sbtcContractIdMainnet = 'SM3VDXK3WZZSA84XXFKAFAF15NNZX32CTSG82JFQ4.sbtc-token::sbtc-token';
+
+export const defaultSbtcLimits = {
+  pegCap: 1000000000000,
+  perDepositCap: 100000000,
+  perDepositMinimum: 100000,
+  perWithdrawalCap: 100000000,
+  accountCaps: {},
+};
+
+const sbtcLimitsResponseSchema = z.object({
+  pegCap: z.number().nullable(),
+  perDepositCap: z.number().nullable(),
+  perDepositMinimum: z.number().nullable(),
+  perWithdrawalCap: z.number().nullable(),
+  accountCaps: z.record(z.any()),
+});
+
+type GetSbtcLimitsResponse = z.infer<typeof sbtcLimitsResponseSchema>;
+
+async function getSbtcLimits(apiUrl: string): Promise<GetSbtcLimitsResponse> {
+  const resp = await axios.get(`${apiUrl}/limits`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
```

Comment:
You don't need to set the content type I'm pretty sure?

Reply:
I did have to add this in the extension originally, but let me check if I can remove it now.